### ISSUE TITLE
composer fresh install: Template not found: web

### ISSUE THREAD

**Issue by lubosdz** [craftcms/cms#2546]
### Description

Following instructions at https://github.com/craftcms/docs/blob/v3/en/installation.md I create new project via `composer create-project -s RC craftcms/craft PATH`. However, accessing base Url `http://localhost/craftcms/web/` throws `Template not found: web`.

Request is routed to `templates/render` with argument `web`. 
However attempting to find template file under default site `D:\_projects\craftcms\templates\default` fails, because subdirectory `default` does not exist, only parent directory `D:\_projects\craftcms\templates\` and is empty.

Since this is my first attempt to setup CraftCms I am not sure whether I missed some installation step or how should I create the template directory with files.

Thank you.

### Steps to reproduce

1. `composer create-project -s RC craftcms/craft craftcms`
2. access site URL `http://localhost/craftcms/web`

### Additional info

- Craft version: 3.0.0-RC14
- PHP version: 7.0.19
- Database driver & version: mysql 5.5.24
- Plugins & versions: -
- OS: **Windows** 7

**Comment by brandonkelly** [comment]
Hey @lubosdz, what happens if you go to http://localhost/craftcms/ instead?

**Comment by lubosdz** [comment]
I reinstalled fresh everything once again to re-check, still same issue.

* when I go to http://localhost/craftcms/ - it will list directories, one of them is `web` (webdav).
* when I go to http://localhost/craftcms/web - I see Exception `404 - Not found` (seeks template `web` in empty directory `craftcms/templates/`).

I have also setup apache virtual host `http://craftcms.local` with webroot `d:\_projects\craftcms\web\` - still 404 - Not found - same issue.

So I downloaded Happy Lager from github, copied `app` into `templates/*.*` and `assets` into `web/assets/*.*` - now I get Exception from `D:\_projects\craftcms\templates\_layouts\site.html` line 59:
`Twig Runtime Error` - Neither the property "environmentVariables" nor one of the methods "environmentVariables()", "getenvironmentVariables()"/ "isenvironmentVariables()"/ "hasenvironmentVariables()" or "__call()" exist and have public access in class "craft\web\twig\variables\Config".

During CLI site setup I entered:
Site URL: **http://localhost/craftcms**
which I am not sure about ..

Sorry for bothering, I know you have bigger issues than my local setup, I will figure out somehow, just wanted to point out the issue. 

BTW Craft should also allow loading templates/views as plain `*.php` files. We don't favour using twig since it adds on to learning curve and brings additional parsing issues. We prefer simple PHP files just like Yii/Yii2 framework, but craftcms seeks by default only for `*.html, *.twig` extensions.

Anyway, it's still most advanced Yii2 CMS so far we have evaluated, great work, thank you.

**Comment by brandonkelly** [comment]
Alright, from what I’m gathering, here’s the good news:

- http://localhost/craftcms is correctly pointed at your `web/` directory.
- http://localhost/craftcms/web is actually working correctly (it’s checking the `templates/` directory for `web.html` `web/index.html`, etc., and returning the correct error as those don’t exist).

I’m guessing the issue is just that your web server isn’t set to serve `index.php` files by default. You can verify by trying to access http://localhost/craftcms/index.php. If that works, you should be able to get it working sans-`index.php` by adding `index.php` to Apache’s [DirectoryIndex](http://httpd.apache.org/docs/2.0/mod/mod_dir.html#directoryindex) directive and restarting the web server.

> BTW Craft should also allow loading templates/views as plain *.php files. We don't favour using twig since it adds on to learning curve and brings additional parsing issues. We prefer simple PHP files just like Yii/Yii2 framework, but craftcms seeks by default only for *.html, *.twig extensions.

Craft gives you the framework for creating a custom Yii module, which can provide its own controller actions, and you can set up routes to them in `config/routes.php`. The controller actions are free to render PHP views instead of Craft templates if you want. See the [modules/Module.php](https://github.com/craftcms/craft/blob/master/modules/Module.php) file for a starting point; or check out https://pluginfactory.io/, which can generate all the scaffolding for a module if you want. (Choose “Craft CMS _Module_ version 3.x” at the top.)

**Comment by lubosdz** [comment]
Thank you for your answer, I did not expect it :-)

There is no `d:\_projects\craftcms\index.php`. The `index.php` file is located under `d:\_projects\craftcms\web\index.php` - created by composer. 

Apache `DirectoryIndex` is set correctly I guess to seek for index.php, index.html and index.htm. 

Here's structure created after setting up `craftcms/craft` project via composer:

```
- craftcms
   - config
      - redactor
   - modules
   - storage
      - logs
   - templates
      .gitignore
   - vendor
      - ... many dirs ...
   - web
      - cpresources
      index.php
      .htaccess
.env
composer.lock
craft.bat
craft
composer
README.md
LICENSE.md
.env.example

```

My setup might be somehow locally specific. 
Let's see if there are more users having same issue when setting up Craft3 ...
Thanx a lot.

**Comment by brandonkelly** [comment]
> There is no `d:\_projects\craftcms\index.php`. The `index.php` file is located under `d:\_projects\craftcms\web\index.php` - created by composer. 

Gotcha. In that case the issue is that you set the wrong site URL in the setup – it should have been http://localhost/craftcms/web.

Easy enough to fix. You should be able to access the Control Panel at http://localhost/craftcms/web/admin. From there navigate to Settings → Sites → [your site] and add `/web` to its base URL. Or you can add a `siteUrl` config setting to config/general.php with the correct URL, and that will override whatever is stored in the database.

**Comment by lubosdz** [comment]
Aah, yes, that worked! 
After I changed `Base URL` to `@web` in Sites -> Settings and also created one plain file under `templates/index.html`.

Perhaps it would be:
* less error-prone to preset `@web` for Base URL during CLI setup
* include one simple plain `Hello World` message in file `templates/index.html`

(I assume that craftcms/craft is actually a plain empty project just to setup directory structure, without any files - so actually that confusing 404 - not found exception - will show up to everybody, though technically it's not an error. But for me playing first time with Craft it looks like I did something wrong ... Just to show Hello message + minimum instructions would be less confusing).

Thank you very much for your help.
